### ISSUE TITLE
can integration test be performed for manual deploying of services ?

### ISSUE THREAD

**Issue by dipshirajput** [edgexfoundry/edgex-taf#588]
![Screenshot from 2022-03-08 11-23-43](https://user-images.githubusercontent.com/52161957/157175302-57e04d54-63f2-4302-a21e-d6dff0d69487.png)
Hi @cloudxxx8 , @cherrycl , @weichou1229 ,
the screenshot attached is of the document provided for TAF, which says that ,
```
**```
Run Integration testing:

Only support deploying edgex services through docker-compose file.
```**
```

what are we supposed to do if we are deploying the services manually ? other than writing scripts for , restart-services.sh , shutdown.sh , startup.sh , stop-services.sh.

**Comment by cloudxxx8** [comment]
@dipshirajput I open an issue for the future improvement
https://github.com/edgexfoundry/edgex-taf/issues/589

You can't run integration test in the non-docker environment without resolving it.

**Comment by jumpingliu** [comment]
@dipshirajput 
The problem has been solved in #589, so close this issue.
